_extras/snippets_library/EPCC_Cirrus_pbs/14/depend_demo.snip

```diff
@@ -0,0 +1,47 @@
+To demonstrate, let's load the `abinit` module and then use the `module list` command to show
+which modules we currently have loaded in our environment. ([Abinit](https://www.abinit.org/) is an open source 
+materials science modelling software package.)
+
+```
+[yourUsername@cirrus-login0 ~]$ module load abinit
+[yourUsername@cirrus-login0 ~]$ module list
+```
+{: .bash}
+```
+Currently Loaded Modulefiles:
+  1) anaconda/python3                  5) intel-compilers-17/17.0.2.174     9) netcdf/4.4.1
+  2) mpt/2.16                          6) intel-cmkl-17/17.0.2.174         10) abinit/8.2.3-intel17-mpt214
+  3) intel-cc-17/17.0.2.174            7) gcc/6.2.0
+  4) intel-fc-17/17.0.2.174            8) fftw-3.3.5-intel-17.0.2-dxt2dzn
+```
+{: .output}
+
+So in this case, loading the `abinit` module
+also loaded a variety of other modules.
+Let's try unloading the `abinit` package.
+
+```
+[yourUsername@cirrus-login0 ~]$ module unload abinit
+[yourUsername@cirrus-login0 ~]$ module list
+```
+{: .bash}
+```
+Currently Loaded Modulefiles:
+  1) anaconda/python3
+```
+{: .output}
+
+So using `module unload` "un-loads" a module along with it's dependencies.
```

Comment:
"it's" -> "its"

Reply:
@tkphd Fixed in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/579a4bdd87c290832744133096f211a6555b24bb